internal/testdata/expected/javascript/ir/statements.js.out

```diff
@@ -178,27 +178,29 @@ func SwitchStatement():
 0:                                                                         entry
 	%t0 = console.log("switch entry")
 	%t1 = "Oranges"
-	%t4 = %t1 == "Oranges"
-	if %t4 goto 3 else 4
+	%t2 = %t1 == "Oranges"
+	if %t2 goto 2 else 3
 1:                                                                       if.done
 	%t9 = console.log("switch done")
-2:                                                                       if.else
-	%t2 = console.log("switch case default")
-	jump 1
-3:                                                                       if.then
+2:                                                                       if.them
```

Comment:
I thkink that this should be `then`, `them` is used for plural like `they`.

Reply:
Done